lib/winston/config.js

```diff
@@ -6,6 +6,9 @@
  *
  */
 
+// Fix colors not appearing in non-tty environments
+require('colors').enabled = true;
```

Comment:
This should be `require('colors/safe').enabled = true` to avoid `String.prototype` modifications. Actually, adding it on line 13 with just `colors.enabled = true` should be more than enough.